test/support/commands.js

```diff
@@ -28,6 +28,23 @@ Cypress.Commands.add('drawRegion', ({ x = 200, y = 200, width = 100, height = 10
         .trigger('mouseup');
 });
 
+Cypress.Commands.add('selectText', ({ page = 0, block = 0 } = {}) => {
+    cy.get('.textLayer')
+        .eq(page)
+        .children()
+        .eq(block)
+        .trigger('mousedown')
+        .then($el => {
+            const el = $el[0];
+            const document = el.ownerDocument;
+            const range = document.createRange();
+            range.selectNodeContents(el);
+            document.getSelection().removeAllRanges();
```

Comment:
Does `document.getSelection()` need to be called separately or can we store the `selection` in a variable?